controllers/release/adapter.go

```diff
@@ -251,6 +251,22 @@ func (a *adapter) EnsureReleaseIsProcessed() (controller.OperationResult, error)
 		return controller.ContinueProcessing()
 	}
 
+	pipelineRuns, err := a.loader.GetActiveManagedReleasePipelineRuns(a.ctx, a.client, a.release)
+	if err != nil {
+		return controller.RequeueWithError(err)
+	}
+
+	// Requeue de Release if other PipelineRun is running for the same Application
+	if len(pipelineRuns.Items) > 0 {
+		patch := client.MergeFrom(a.release.DeepCopy())
+		a.release.MarkReleaseQueued(fmt.Sprintf("%d Release(s) running for the same Application", len(pipelineRuns.Items)))
```

Comment:
What's running is not releases, but pipelineruns. Maybe "Release PipelineRuns" would work?